### DISCUSSION TITLE
Enable kernel network stack path for Shortcut Forwarding Engine？

### DISCUSSION THREAD

**Discussion by devnakx** [Boos4721/openwrt#93] [Bugs/问题反馈]
![1](https://user-images.githubusercontent.com/38558065/158363205-0cabfc4a-bb1c-4b8a-b852-c6df5dc60a68.png)
最近源码编译到一半会问这个，虽然可以手动输入N/y继续，但是有时候挂机编译回来一看卡在这了就挺影响的

**Comment by Boos4721** [comment]
FIXED
